.github/workflows/test.yml

```diff
@@ -216,46 +216,6 @@ jobs:
         fi
 
 
-    # Ensure tests pass on oldest supported Go version.
-    # Do while inside GOPATH, which tests different parts
-    # of the code. This is panicparse specific and will be removed after 2022.
-  old:
-    name: "test: go${{matrix.gover}}/${{matrix.os}}"
-    runs-on: "${{matrix.os}}"
-    continue-on-error: true
-    defaults:
-      run:
-        shell: bash
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu-latest]
-        # https://github.com/golang/go/issues/55078
-        # golang.org/x/sys/unix broke on Go versions before 1.17. Not worth
-        # fixing.
-        gover: ['1.17.13']
```

Comment:
Maybe bump it to 1.21 instead? I like running on more than one version. I understand it it maybe be too annoying due to the glob/init change.

Reply:
Yeah, I didn’t include any other versions because we’d need to fork the examples. If you want to do that, would you mind doing it in a follow-up change independent of this test fix?

Reply:
What about you keep it and set it to 1.23.6? This way we can at least keep the workflow.